profile/README.md

```diff
@@ -18,5 +18,7 @@ If you would like to be more up to date on the progress within Pulsar feel free
 * [Tentative Roadmap Discussion](https://github.com/orgs/pulsar-edit/discussions/2)
 * [Frequently Asked Questions](https://github.com/pulsar-edit/.github/blob/main/FAQ.md)
 * [Contributing Right Now](https://github.com/pulsar-edit/.github/blob/main/project-birth/CONTRIBUTING-DURING-START.md)
+* [Documentation Roadmap](https://github.com/pulsar-edit/pulsar-edit.github.io/issues/14)
+* [Hop Right In on Some PR's or Issues](https://github.com/pulls?q=is%3Aopen+user%3Apulsar-edit+archived%3Afalse+sort%3Acomments-asc)
```

Comment:
```suggestion
* [Hop Right on in with some PR's and Issues](https://github.com/pulls?q=is%3Aopen+user%3Apulsar-edit+archived%3Afalse+sort%3Acomments-asc)
```
Just some logistical edits, because this sounded a bit odd originally in my opinion.

Reply:
Good catch, don't know where that grammar or capitalization came from